# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/crashtests.list
# Commit: 0825e43199d2
# Full Hash: 0825e43199d20888d87c3038413cdb652712a275
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-07-16 04:08:30
# Description:
#   Bug 1907464 - Clear the shadow-crossing selection when the ParentChainChanged notification is called on the common ancestor r=jjaschke,dom-core
#   
#   It's unclear from the spec about where the new selection should be,
#   so we just clear the selection at the moment.
#   
# ==============================================================================

diff -r 399b1edeca46 -r 0825e43199d2 dom/base/crashtests/crashtests.list
--- a/dom/base/crashtests/crashtests.list	Mon Jul 15 15:20:11 2024 +0000
+++ b/dom/base/crashtests/crashtests.list	Mon Jul 15 15:29:05 2024 +0000
@@ -280,3 +280,4 @@
 load 1890899.html
 load 1896225.html
 load 1897248.html
+load 1907464.html
